hack/manifests/backstage/backstage.yaml

```diff
@@ -26,7 +26,7 @@ spec:
             - "packages/backend"
             - "--config"
             - "/app/app-config.yaml"
-          image: quay.io/parodos-dev/backstage-parodos:latest
+          image: quay.io/parodos-dev/backstage-parodos:latest-openshift
```

Comment:
Are there any changes how we build the image specific to ocp?

Reply:
As agreed, let's have this in and check whether we can use the same image everywhere